eosiojava/src/main/java/one/block/eosiojava/models/rpcProvider/ContextFreeData.java

```diff
@@ -0,0 +1,118 @@
+package one.block.eosiojava.models.rpcProvider;
+
+import java.nio.ByteBuffer;
+import org.bitcoinj.core.Sha256Hash;
+import org.bouncycastle.util.encoders.Hex;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.List;
+
+public class ContextFreeData {
+    @NotNull
+    public List<String> data;
+
+    @NotNull
+    public byte[] rawBytes;
+
+    public ContextFreeData(@NotNull List<String> contextFreeData) {
+        this.setData(contextFreeData);
+    }
+
+    @NotNull
+    public List<String> getData() {
+        return this.data;
+    }
+
+    @NotNull
+    public byte[] getBytes() {
+        return this.rawBytes;
+    }
+
+    public void setBytes(byte[] bytes) {
+        this.rawBytes = bytes;
+    }
+
+    public void setData(List<String> contextFreeData) {
+        this.data = contextFreeData;
+        if (!this.hasData()) {
+            this.setBytes(new byte[0]);
+            return;
+        }
+
+        ByteBuffer buffer = ByteBuffer.allocate(this.getTotalBytes(this.data));
```

Comment:
I could see this `ByteBuffer` logic split out to a separate utils class, but I left it here since we decided to not push this logic to the `SerializationProvider`.

Reply:
I think it's fine here.  Can you add some JavaDoc and comments around the class and public methods so it's clear on how these methods are to be used?  (Ex.  What is the ContextFreeData class used for?). For format, please use some of the other models as reference.

Reply:
Updated.